mappings/net/minecraft/loot/entry/TagEntry.mapping

```diff
@@ -1,7 +1,19 @@
 CLASS net/minecraft/unmapped/C_icycuopn net/minecraft/loot/entry/TagEntry
 	FIELD f_hgeahzgw expand Z
 	FIELD f_qxtjppyb name Lnet/minecraft/unmapped/C_ednuhnnn;
-	METHOD m_gedwawmu builder (Lnet/minecraft/unmapped/C_ednuhnnn;)Lnet/minecraft/unmapped/C_sphmzmsf$C_lbxiirfw;
+	METHOD <init> (Lnet/minecraft/unmapped/C_ednuhnnn;ZII[Lnet/minecraft/unmapped/C_vqkczpuv;[Lnet/minecraft/unmapped/C_tmxhskql;)V
+		ARG 1 name
+		ARG 2 expand
+		ARG 3 weight
+		ARG 4 quality
+		ARG 5 conditions
+		ARG 6 functions
+	METHOD m_gedwawmu tagContents (Lnet/minecraft/unmapped/C_ednuhnnn;)Lnet/minecraft/unmapped/C_sphmzmsf$C_lbxiirfw;
+		ARG 0 name
+	METHOD m_loigdgvv (Ljava/util/function/Consumer;Lnet/minecraft/unmapped/C_cjzoxshv;)V
+		ARG 1 name
+	METHOD m_qvbpbjtq expandTag (Lnet/minecraft/unmapped/C_ednuhnnn;)Lnet/minecraft/unmapped/C_sphmzmsf$C_lbxiirfw;
```

Comment:
I'm a little unclear on what this does, but I think either this name or the name of what's currently `tagContents` should be changed to reflect that they're identical except for the `expend` arg. 
So if `tagContents` is left as-is, this would be something like `expendedTagContents`; 
or if `tagContents` changes to `ofTag`, this would be something like `ofExpandedTag`.